src/Discord.Net.Rest/API/Common/Guild.cs

```diff
@@ -66,7 +66,7 @@ internal class Guild
         [JsonProperty("premium_subscription_count")]
         public int? PremiumSubscriptionCount { get; set; }
         [JsonProperty("preferred_locale")]
-        public string PreferredLocale { get; set; }
+        public Optional<string> PreferredLocale { get; set; }
```

Comment:
docs doesnt state this as being optional. doesnt it default to "en-US"?

Reply:
Yes, you are correct.
And I reverted this change since this was not needed anyway. this was my oversight